content/custom-commands/commands.md

```diff
@@ -108,19 +114,24 @@ Delete command button
 
 ### ID and Name
 
-Custom Commands are identified by either their ID or their name.
+Custom commands are identified by either their ID or their name.
+
+When a custom command is created, it is assigned a numeric **ID** (**1**) starting at `1`. The number increases with
+each custom command created on your server. It is not based on the _current_ number of custom commands, but the total
+commands that have ever been created on the server. IDs cannot be changed by the user.
 
-When a custom command is created, the system assigns it a numeric **ID** (**1**) starting at `1`. The number increases
-with each custom command created on your server. It is not based on the _current_ number of custom commands, but the
-total commands that have ever been created on the server. This ID is permanent, and cannot be changed or re-ordered.
+The ID uniquely identifies a custom command, and is therefore used in a variety of contexts where one needs to supply a
+specific custom command. For instance, the `execCC` custom command function targets a specific CC ID, and some built-in
+commands like `customcommands` accept a CC ID as an argument.
 
-The ID may be used to identify a CC for a variety of purposes. Calling `{{ .CCID }}` within a command response will
-return its ID. You must target a CC ID for `execCC`. You can use run the `CustomCommands` command with a CC ID to
-retrieve info about that CC.
+Within a command response, the ID may be retrieved using the `{{ .CCID }}` template.
 
 {{% notice warning %}}
 
-Deleting a Custom Command does not allow its ID to be reassigned. If you delete a CC, its ID is lost forever.
+Custom command IDs cannot be changed by the user in any way, save by deleting and recreating a command.
+
+Note, however, that deleting a custom command does not allow its ID to be reassigned. That is, even if you delete a CC,
+its ID will not be reused.
```

Comment:
This seems more confusing imo

Reply:
I agree after rereading, good call. Will revert.